src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -24,6 +26,16 @@ export class CalciteShellPanel {
    */
   @Prop({ reflect: true }) collapsed = false;
 
+  /**
+   * floatContent lets the content area appear like a floating panel and only sets a max-height.
+   */
+  @Prop({ reflect: true }) floatContent = false;
```

Comment:
Is there a name for this that doesn't sound like an action/verb? On a related note, is there an alternate word we could use for `float` to avoid any possible confusion w/ CSS floats?

Reply:
@pr3tori4n Had the same concern and suggested `detached`. Maybe `contentDetached`?